pkg/securestorage/securestorage.go

```diff
@@ -63,30 +110,30 @@ func openKeyring() (keyring.Keyring, error) {
 		return nil, err
 	}
 
-	credStorePath := path.Join(grantedFolder, "cred-store")
-
+	secureStoragePath := path.Join(grantedFolder, "secure-storage-"+s.StorageSuffix)
+	name := "granted-" + s.StorageSuffix
```

Comment:
Note that renaming this will cause users to have to allow granted access to the keychain or setup a new secure password

Reply:
It was required to be able to have 2 seperate stores for credentials and sso tokens